eng/Directory.Packages.props

```diff
@@ -5,7 +5,7 @@
     <MicrosoftCodeAnalysisNetAnalyzersVersion>8.0.0-preview.23468.1</MicrosoftCodeAnalysisNetAnalyzersVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.1.3-beta1.24319.1</MicrosoftCodeAnalysisTestingVersion>
     <MicrosoftVisualStudioExtensibilityTestingVersion>0.1.796-beta</MicrosoftVisualStudioExtensibilityTestingVersion>
-    <_BasicReferenceAssembliesVersion>1.7.9</_BasicReferenceAssembliesVersion>
+    <_BasicReferenceAssembliesVersion>1.8.3</_BasicReferenceAssembliesVersion>
```

Comment:
Should we add `TargetFramework.Net100`?
Then we can add a simple test verifying the presence of expected new API in .NET 10.